### PR TITLE
#831 Add renaming from supplier requisition to request requisition

### PR DESCRIPTION
**Pull request by joshxg**
#831 - renaming supplier requisition -> request requisition / internal orders

- In the UI it's internal orders
- internal stuff is all request requisition

I don't know if correct, but it's done now

### CODE REVIEW

File: packages/common/src/intl/locales/en/app.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
   "cmdk.create-inbound": "Create: New Inbound Shipment",
   "cmdk.goto-dashboard": "Go to: Dashboard",
   "cmdk.goto-customer-requisition": "Go to: Customer Requisition",
-  "cmdk.goto-supplier-requisition": "Go to: Supplier Requisition",
+  "cmdk.goto-internal-order": "Go to: Internal orders",
```
Comment (mark-prins) [inline packages/common/src/intl/locales/en/app.json:44]: we only have 'customer requisition' in `CommandK.tsx` - might be time to add a goto for `internal orders` too

File: packages/requisitions/src/CustomerRequisition/DetailView/Footer.tsx [hunk 2]
```diff
@@ -9,11 +9,11 @@ import {
 } from '@openmsupply-client/common';
 import React, { FC } from 'react';
 import {
-  getSupplierRequisitionStatuses,
-  getNextSupplierRequisitionStatus,
-  getSupplierRequisitionTranslator,
+  getRequestRequisitionStatuses,
```
Comment (mark-prins) [inline packages/requisitions/src/CustomerRequisition/DetailView/Footer.tsx:12]: just curious / confused. Is this due to the reuse of status across both request and response requisitions?
We're in the `CustomerRequisition` directory using the `RequestRequisitionStatus`

Comment (mark-prins) [reply packages/requisitions/src/CustomerRequisition/DetailView/Footer.tsx:12]: Oh, I see that there's a `RequisitionNodeStatus` later on.. maybe copy paste from supplier reqs then?

**Review by mark-prins — APPROVED** [review]
👍 
Thanks - much better! I do a little mental juggling with the `internal order` == `request requisition` still.. though you've got the separation nicely covered. I think it's pretty clear

File: packages/requisitions/src/CustomerRequisition/DetailView/Footer.tsx (hunk 2 above)
Comment (joshxg) [reply packages/requisitions/src/CustomerRequisition/DetailView/Footer.tsx:12]: Sorry - have made this for spending some more time on statuses https://github.com/openmsupply/openmsupply-client/issues/849

File: packages/common/src/intl/locales/en/app.json (hunk 1 above)
Comment (joshxg) [reply packages/common/src/intl/locales/en/app.json:44]: ah yes, thanks

**Pull request merged**